src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java

```diff
@@ -526,37 +502,37 @@ public void testClearExistingCache() throws Exception {
         final String[] expected = CommonUtils.EMPTY_STRING_ARRAY;
 
         verify(checker, pathToEmptyFile, pathToEmptyFile, expected);
+
         checker.clearCache();
```

Comment:
why call `verify` if we are going to clear cache right after before checking anything?